### DISCUSSION TITLE
Component doesn't render when back/forward browser history using Query String

### DISCUSSION THREAD

**Discussion by kurowaffle** [livewire/livewire#5638] [1. Help]
Hello.

I made a simple full-page component with a simple query string.

Controller
```
class LivewireTest extends Component
{
    public $activeMenu = '';

    protected $queryString = [
        'activeMenu' => ['except' => '', 'as' => 'type'],
    ];

    public function render()
    {
        return view('testbaru::livewire-test')->layout('testbaru::livewire-layout');
    }

    public function ChangeMenu($menu)
    {
        $this->activeMenu = $menu;
    }
}
```

Component `livewire-test.blade.php`
```
<div>
    <div wire:key="{{ $this->activeMenu }}">
        <div wire:loading>
            Loading ...
        </div>
        <br>
        <button type="button" wire:click="ChangeMenu('home')">Menu</button>
        <button type="button" wire:click="ChangeMenu('profile')">Profile</button>
        <br>
        @if ($activeMenu == "home")
            Home Active
        @else
            Profile Active
        @endif
    </div>
</div>
```

Layout `livewire-layout.blade.php`
```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Document</title>
    @livewireStyles
</head>
<body>
    <div>
        Welcome!
        <br>
        <br>
        {{ $slot }}
    </div>

    @livewireScripts
</body>
</html>
```

Component changed successfully when I click the buttons, but didn't work when I click back button in browser. I'm using Google Chrome.
Here the detail: [https://gifyu.com/image/SIUsh](https://gifyu.com/image/SIUsh)
Am I doing wrong? Or this is a bug?

Thank you

**Comment by joshhanley** [comment]
@kurowaffle hmm thats strange, I'd expect that to work.

Have you by any chance published Livewire's config and enabled the back button cache?

If so, try disabling it again, as there are issues with Livewire and the back button cache. It should be disabled by default though if you haven't touched it.

https://github.com/livewire/livewire/blob/019b1e69d8cd8c7e749eba7a38e4fa69ecbc8f74/config/livewire.php#L127-L141

Hope this helps!
